@xen-orchestra/proxy/src/app/mixins/backups/index.js

```diff
@@ -46,26 +46,14 @@ export default class Backups {
       await fromCallback(execFile, 'pvscan', ['--cache'])
     })
 
-    let run = ({ recordToXapi, remotes, xapis, ...rest }) =>
-      new Backup({
-        ...rest,
-
-        // don't change config during backup execution
-        config: app.config.get('backups'),
-
-        // pass getAdapter in order to mutualize the adapter resources usage
-        getAdapter: remoteId => this.getAdapter(remotes[remoteId]),
-
-        getConnectedRecord: Disposable.factory(async function* getConnectedRecord(type, uuid) {
-          const xapiId = recordToXapi[uuid]
-          if (xapiId === undefined) {
-            throw new Error('no XAPI associated to ' + uuid)
-          }
-
-          const xapi = yield this.getXapi(xapis[xapiId])
-          return xapi.getRecordByUuid(type, uuid)
-        }).bind(this),
-      }).run()
+    let run = (params, onLog) =>
+      runBackupWorker(
```

Comment:
If not too much difficult, I think it would be nice to if it could be optional to run in a worker.

We'll discuss about this.

Reply:
Optional using the API or inside the proxy?

Reply:
Inside the proxy and `xo-server`, the ability to not use the workers, in case there are unforeseen side-effects for some users.

Reply:
A configurable property which deny the usage of the workers?

Reply:
Why not, a simple config entry to disable the workers.